mmdet/models/backbones/resnet.py

```diff
@@ -235,19 +278,17 @@ def __init__(self,
         assert max(out_indices) < num_stages
         self.style = style
         self.frozen_stages = frozen_stages
-        self.bn_eval = bn_eval
-        self.bn_frozen = bn_frozen
+        assert (isinstance(normalize, dict) and 'eval_mode' in normalize
+                and 'frozen' in normalize)
+        self.norm_eval = normalize.pop('eval_mode')
```

Comment:
There is a bug for `normalize.pop('eval_mode')`. We need to first copy the variable `normalize`, otherwise it will raise an error when we call the init function more than once.
```python
model = ResNet(50)
model = ResNet(50)  # will raise an error
```

Reply:
Fixed in [82e7545](https://github.com/open-mmlab/mmdetection/pull/160/commits/82e75455fda13d3273b8a8d23ebf34d9f10e1ed8)